INSTALL.rst

```diff
@@ -86,7 +86,7 @@ b. Debian and Ubuntu
 ````````````````````
 Install all the required dependencies::
 
-  sudo apt-get install python-matplotlib python-numpy python-pil python-scipy
```

Comment:
I was actually hoping to trigger this discussion.
Since `scikit-image` project requires Python >=3.5 then in my opinion it would be advisable to use `python3-x` alternatives. 
I'm happy to do it here.

Reply:
yeah, that should be done. I just stopped using system python, because I was stuck on 16.04 for a while.

Reply:
@hmaarrfk fixed, please review again

Reply:
should cython be updated below?

Reply:
apparently should as well. done.

Reply:
Cool awesome.

Reply:
@hmaarrfk In 18.04 `python` is still `python2`.

Reply:
i guess not! i guess I was stuck on 16.04 for too long, but i think the package names are still python2 package names. ipython will point to python2.

Reply:
I'm on Ubuntu 18.04. On the example of `cython3` you can see how it relates to Python version. Same happens for all the other packages I changed in this PR. 
3: 
```
agata@agata-XPS-15-9550:~$ apt-cache show cython3
Package: cython3
Architecture: amd64
Version: 0.26.1-0.4
Priority: optional
Section: universe/python
Source: cython
Origin: Ubuntu
Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Original-Maintainer: Python Applications Packaging Team <python-apps-team@lists.alioth.debian.org>
Bugs: https://bugs.launchpad.net/ubuntu/+filebug
Installed-Size: 8588
Depends: python3 (<< 3.7), python3 (>= 3.6~), python3:any (>= 3.3.2-2~), libc6 (>= 2.14)
Recommends: python3-dev, gcc
...
```
no number (defaults to 2):
```
agata@agata-XPS-15-9550:~$ apt-cache show cython
Package: cython
Architecture: amd64
Version: 0.26.1-0.4
Priority: optional
Section: universe/python
Origin: Ubuntu
Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Original-Maintainer: Python Applications Packaging Team <python-apps-team@lists.alioth.debian.org>
Bugs: https://bugs.launchpad.net/ubuntu/+filebug
Installed-Size: 8439
Depends: python (<< 2.8), python (>= 2.7~), python:any (<< 2.8), python:any (>= 2.7.5-5~), libc6 (>= 2.14)
```

Reply:
I was telling the same :) (that Python defaults to 2.7 in the latest LTS).
Thanks for the PR!

Reply:
Oh, of course, my english is just deteriorating day by day.